Services/UICore/lib/html-it/IT.php

```diff
@@ -1,12 +1,26 @@
 <?php
 
+/**
+ * This file is part of ILIAS, a powerful learning management system
+ * published by ILIAS open source e-Learning e.V.
+ *
+ * ILIAS is licensed with the GPL-3.0,
+ * see https://www.gnu.org/licenses/gpl-3.0.en.html
+ * You should have received a copy of said license along with the
+ * source code, too.
+ *
+ * If this is not the case or you just want to try ILIAS, you'll find
+ * us at:
+ * https://www.ilias.de
+ * https://github.com/ILIAS-eLearning
+ *
+ *********************************************************************/
+
 declare(strict_types=1);
 
 // +----------------------------------------------------------------------+
-// | Copyright (c) 1997-2005 Ulf Wendel, Pierre-Alain Joye                |
 // +----------------------------------------------------------------------+
 // | This source file is subject to the New BSD license, That is bundled  |
-// | with this package in the file LICENSE, and is available through      |
```

Comment:
@chfsx I believe these two lines (6, 9) have been removed by the rector automatically. I'm no expert in copyright-law, so I have no idea if we need to leave the old copyright in here, since we have made some adjustments to the file. I also don't know if we can just put our ILIAS license header in there, since this file has not been written by ILIAS initially. I suggest to add the removed lines back in at least.